docs/how-to/rocm-for-ai/accelerate-training.rst

```diff
@@ -0,0 +1,130 @@
+***************************
+Accelerating model training
+***************************
+
+To train a large model like GPT2 or Llama 2 70B, a single accelerator or GPU cannot store all the model parameters
+required for training. What if you could convert the single-GPU training code to run on multiple accelerators or GPUs?
```

Comment:
"What if you could"
Perhaps "Could you"

Reply:
Updated